geomesa-tools/bin/download-data.sh

```diff
@@ -53,14 +53,16 @@ case "$type" in
   tdrive)
 
     echo "Note: each zip file contains approximately one million points"
-    read -p "Download how many zip files? " NUM
+    read -p "Download how many zip files? (9 total) " NUM
```

Comment:
When I downloaded it a few months ago, there were zips 01-014. But is seems that 01-05 are no longer hosted...
That page doesn't render fully for me in chrome, either.